src/black/__init__.py

```diff
@@ -1261,6 +1263,11 @@ def get_features_used(  # noqa: C901
             value_head = n.value[:2]
             if value_head in {'f"', 'F"', "f'", "F'", "rf", "fr", "RF", "FR"}:
                 features.add(Feature.F_STRINGS)
+                if Feature.DEBUG_F_STRINGS not in features:
+                    for span_beg, span_end in iter_fexpr_spans(n.value):
```

Comment:
Is this an expensive function, or is its speed negligible? If it is a bit slow, we could also add a more naive check above ("=" in n.value) to only include f-strings with equal signs (which should reduce the number of calls to this function by a lot). If it is not too expensive, I think it is alright to keep it as is.

Reply:
It is not too expensive, it's a single loop over the value. mypyc should also compile it well.

Reply:
Cool! Then no need.